migrations/helpers/shared_helper.py

```diff
@@ -81,16 +81,13 @@ def get_cherrypicked_samples(
         ]
 
         sql_engine = sqlalchemy.create_engine(
-            (
-                f"mysql+pymysql://{config.MLWH_DB_RO_USER}:{config.MLWH_DB_RO_PASSWORD}"  # type: ignore
-                f"@{config.MLWH_DB_HOST}"  # type: ignore
-            ),
+            (f"mysql+pymysql://{config.MLWH_DB_RO_USER}:{config.MLWH_DB_RO_PASSWORD}" f"@{config.MLWH_DB_HOST}"),
```

Comment:
is this the bit we added the port to when doing the migration? Might need careful merging

Reply:
Indeed, will add the port here to make doubly sure too.